virtool/authorization/api.py

```diff
@@ -14,21 +14,18 @@
 
 @routes.view("/source/permissions")
 class SourceView(PydanticView):
-    async def get(
-        self, resource_type: Optional[ResourceType] = None
-    ) -> r200[ListPermissionsResponse]:
+    async def get(self) -> r200[ListPermissionsResponse]:
         """
         List permissions.
 
         Lists all Virtool permissions.
 
-        The list can be filtered by resource type using the
-        `resource_type` query parameter.
+        The list can be filtered by resource type using the `resource_type` query
+        parameter.
 
         Status Codes:
             200: Successful operation
         """
-
         return json_response(
             await get_data_from_req(self.request).auth.find(resource_type)
```

Comment:
```suggestion
            await get_data_from_req(self.request).auth.find()
```
Remove resource_type and update testing since it is not being used for now.